### PR TITLE
Created online docs, fixed div class issue used to scrape lyrics

### PR DESCRIPTION
**Pull request by Allerter**
### Documentation
Have a look at the docs [here](https://lyricsgenius.readthedocs.io/en/latest/index.html)
Considering #63 and the need for docs, I created documentation for the library using _Sphinx_ and also used the RTD theme for it. I used the Google style for documenting the classes, and the functions. Since the library supports all Python 3 versions, I didn't use type hints to auto-document things in Sphinx and added them manually in the docstrings. I also added some examples under the methods, and some more are in the snippets page of the docs.
I also added a picture to the index page for aesthetics. I didn't use the Genius logo, because I wasn't sure if using their logo here was allowed or no.
This is the first version of the docs, so I'd appreciate any suggestions.
To check out the built version of the docs for yourself, install the packages in the requirements file in the docs folder. And then run `make html` in the same directory (or just use the tox commands below).
### PEP8 Check
I also did the PEP8 checks, but I'm not sure if all of them contributed to the clarity and style of the code. So feel free to revert any of those. For PEP8, I used _flake8_ coupled with _flake8-bugbear_. As for the configs I set for flake8, they're all present in the tox.ini file.
### The dreaded DIV class
Looks like Genius went and changed up the class of the lyrics' div tag again (#148). I provided two solutions in the mentioned issue, but two users reported that it didn't work for them. But I can't really say anything about that since the fixes both work for me. So I added the second fix to the code.
### Tox
I added tox so you can check things yourself. Running `tox` in the source dir of the library will run flake8, doc8 (PEP8 for docs), and create the docs to see if they are created correctly. You can run the following tox commands for the checks:

- `tox` - runs flake8, doc8, and tests creating docs
- `tox -e lint` - rung flake8 and doc8
- `tox -e docs` tests creating docs
- `tox - test` runs the tests using pytest (needs GENIUS_CLIENT_ACCESS_TOKEN env var set)

For tox, I had to edit setup.py, add some things to it, and remove reading the requirements from the text file. But if you don't want to keep the tox file, feel free to revert setup.py and remove the tox file.

### CODE REVIEW

**Comment by Allerter** [comment]
The home page is a little short on content. The text of [Scraping song lyrics from Genius.com 🎶](https://www.johnwmillr.com/scraping-genius-lyrics/) could help.

**Comment by johnwmillr** [comment]
Hi @Allerter, is the `tox.ini` file included in your changes here? I have `.ini` files excluded with `gitignore`, so you may have unknowingly not added it.

**Comment by Allerter** [comment]
I wasn't aware that `.ini` files were excluded. The `tox.ini` file contains tests for checking the codes against PEP8, checking the docs against PEP8, and checking to create the docs. It can come in handy when someone else wants to make a change in those files. But you can delete it if you want to.

**Comment by johnwmillr** [comment]
Yes, I'm happy to include the `tox.ini` file, but I don't think you've added it to your PR. Add it with a new commit, and I'll include it.

**Comment by Allerter** [comment]
My bad, now I get what you meant. I'll add it in a bit.

File: docs/src/examples/snippets.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.. _snippets:
+
+Snippets
+==================
+Here are some snippets showcasing how the library can be used.
```
Comment (johnwmillr) [inline docs/src/examples/snippets.rst:5]: This is great! Thanks for adding these.

**Comment by Allerter** [comment]
Hey. The checks passed on my Travis CI account. Looks like you haven't set the `GENIUS_CLIENT_ACCESS_TOKEN` in the environment variables of the repository on Travis CI's website.

**Comment by johnwmillr** [comment]
I think I haven't given PRs permission to use my token on Travis. The tests pass in my local environment. I plan to merge your PR this weekend if not before then.

**Comment by Allerter** [comment]
Why not give access when its value is secret?
By the way, I tried to add you as a maintainer on readthedocs.io, but I think you don't have an account there (or it's not the same as your GitHub username). If you'd like to be added, please let me know.

File: docs/src/index.rst [hunk 2]
```diff
@@ -4,14 +4,33 @@
...
+It’s actually not too difficult to start pulling data from the Genius website.
+Genius is hip enough to provide a free application programming interface (API)
+that makes it easy for nerds to programmatically access song and artist data
+from their website. What the Genius API doesn’t provide, however,
+is a way to download the lyrics themselves. With a little help from
+`Beautiful Soup`_ though, it’s possible to grab the song lyrics without too
+much extra work. And LyricsGenius has done all of that for you already.
+
+..
+   source::https://www.johnwmillr.com/scraping-genius-lyrics/
```
Comment (Allerter) [inline docs/src/index.rst:32]: I copied the part above from a post on your website. Is that okay?

**Comment by Allerter** [comment]
I added you on ReadTheDocs. You should have access now.
I'd be glad to take part in maintaining the library and have sent you an email.

**Pull request merged**